src/@types/did.d.ts

```diff
@@ -0,0 +1,24 @@
+declare interface DIDDoc {
```

Comment:
Should we create an [@types](https://www.npmjs.com/~types) module for DID related objects?

Reply:
@simonas-notcat ^^^

Reply:
I think it makes sense. We could have that as part of the did-resolver itself.

Reply:
@mi-xu I think it makes sense to create sepparate @types module